### ISSUE TITLE
Xamarin FontImageSource causing issues in label

### ISSUE THREAD

**Issue by vsfeedback** [xamarin/Xamarin.Forms#7410]
_This issue has been moved from [a ticket on Developer Community](https://developercommunity.visualstudio.com/content/problem/703146/xamarin-fontimagesource-causing-issues-in-label.html)._

---
I am using FontImageSource to load icons in Shell Bottom Tabs. And in various other pages I am loading icons in Label control.

But in Android, once FontImageSource load the font, all the other icons inside Label stops working and shows some character instead of icon, and only works after restarting the app.

But if I comment FontImageSource code and load a png file then there is no such issue.

So it seems there is some bug in FontImageSource.

---
#### Visual Studio Feedback System on 8/22/2019, 00:53 AM: 

<p>We have directed your feedback to the appropriate engineering team for further evaluation. The team will review the feedback and notify you about the next steps.</p>

---
### Original Solutions
(no solutions)

**Comment by samhouts** [comment]
Can you please attach a small project that demonstrates this issue? Thanks!

**Comment by gauravsofts** [comment]
Sample project attached. 

[FontImageSample.zip](https://github.com/xamarin/Xamarin.Forms/files/3609025/FontImageSample.zip)

Thanks,
Gaurav Khanna

**Comment by CliffAgius** [comment]
This looks like a duplicate of #6666 and related #6317 at least on Android they show at app start before giving weird symbols but on IOS they always show the symbols and never the FontIcons.

I am using the MaterialDesignIcons pack so it's not the Font Pack.

**Comment by hartez** [comment]
This looks like some sort of caching issue, or a problem with resources overwriting earlier ones. This works just fine:

```
<StackLayout>
    <Label FontSize="Title" FontFamily="{StaticResource FontAwesomeSolid}" Text="&#xf2b9;"/>
    <Label FontSize="Title" FontFamily="{StaticResource FontAwesomeSolid}" Text="&#xf2b9;"/>
    <Image>
        <Image.Source>
            <FontImageSource FontFamily="{StaticResource FontAwesomeSolid}" Glyph="&#xf2b9;"/>
        </Image.Source>
    </Image>
</StackLayout>
```

But this (note the order) does not:

```
<StackLayout>
    <Label FontSize="Title" FontFamily="{StaticResource FontAwesomeSolid}" Text="&#xf2b9;"/>
    <Image>
        <Image.Source>
            <FontImageSource FontFamily="{StaticResource FontAwesomeSolid}" Glyph="&#xf2b9;"/>
        </Image.Source>
    </Image>
    <Label FontSize="Title" FontFamily="{StaticResource FontAwesomeSolid}" Text="&#xf2b9;"/>
</StackLayout>
```

The second label will display `☒`.

**Comment by hartez** [comment]
... and it looks like this is fixed in 4.3.0.
